src/vtep-internal.c

```diff
@@ -131,40 +130,47 @@ struct vtep_state {
  * @param	dst_ip the destination IP
  * @param	datagram_len the lenght of the datagram including the header
  */
```

Comment:
Missing  "state" parametter in the comment

Reply:
and a pretty much unrelated change anyway
I'm just going to remove this change.